templates/blog/post_list.html

```diff
@@ -0,0 +1,23 @@
+{% extends 'base.html' %}
+
+{% block title %}
+  <title>Blog Posts</title>
+{% endblock %}
+
+{% block content %}
+  <h2>Posts</h2>
+  <div>
+    <ul>
+      {% for post in post_list %}
+        <li>
+          <h3><a href="{% url 'blog:post-detail' pk=post.id %}">{{ post.title }}</a></h3>
```

Comment:
It's a good practice to add the `title` attribute to the link. This attribute is used to provide additional information about the link. It can be beneficial for SEO and accessibility. For example: `<a href="{% url 'blog:post-detail' pk=post.id %}" title="Go to the details of {{ post.title }}">{{ post.title }}</a>`.